services/ui-src/.eslintrc

```diff
@@ -7,6 +7,7 @@
     "plugin:prettier/recommended"
   ],
   "plugins": ["prettier"],
+  "ignorePatterns": ["", "**/public/env-config.js"],
```

Comment:
had three nonsense errors on this file, which is just an object for local env, so i ignored it

(nonsense as in a semicolon on the last item in the object 😵 )